### ISSUE TITLE
uBlock doesn't show a blocked request on the request log

### ISSUE THREAD

**Issue by Mikey1993** [uBlock-LLC/uBlock#519]
<h2>Steps:</h2>

1) Enter: https://www.youtube.com/analytics
2) Observe that the page isn't showing the statistics because of 1 request that is being blocked by some static rule in uBlock - this shows fine in the Dynamic Filtering side bar (with default settings/filter lists).
3) Open up a statistics record for this page via the popup icon to try and identify which request is being blocked.
4) Reload the page for the record.
5) **The blocked request isn't showing.**
6) Try to green (override the static blocked request) of the `youtube.com` domain for the current site from the popup - The analytics page is now loading fine and the whitelisted requests are being shown in the statistics:
`www.youtube.com youtube.com * allow`

**But I still can't figure out what is the request that blocked me from loading the page as it should.**

**Comment by gorhill** [comment]
> Observe that the page isn't showing the statistics because of 1 request that is being blocked
> 
> with default settings/filter lists

I don't get "1 request that is being blocked" with default lists. I get "Loading Analytics..."

**Comment by Mikey1993** [comment]
![1 request blocked](https://i.imgur.com/FpXCPZ1.png)

**Comment by gorhill** [comment]
Ok I can't get what you have in the picture, but there is definitely a problem I saw, which maybe is related to what you are describing. The inline scripts are being blocked on the page because of a static filter in one of the 3rd-party list, while this should definitely not happen, unless the filter option `inline-script` is used.

This is a regression bug since I introduced dynamic filtering: the test to find out whether inline scripts must be blocked has to use `matchStringExactType` not `matchString`. This was fine before I released dynamic filtering.

I need to fix this ASAP, this is a serious bug.
